certbot-dns-cloudxns/setup.py

```diff
@@ -9,7 +9,7 @@
 install_requires = [
     'acme>=0.21.1',
     'certbot>=0.21.1',
-    'dns-lexicon>=2.2.1', # Support for >1 TXT record per name
+    'dns-lexicon>=3.0.6', # Support for >1 TXT record per name
```

Comment:
It's probably safe to remove the ` # Support for >1 TXT record per name` comments now. I don't remember why it was added, but it doesn't seem useful at this point, and almost certainly isn't meaningful following this upgrade.

Reply:
Yes, it corresponds to the version where the support of multiple txt records for the same name has been formerly added in Lexicon (I mean with dedicated tests to ensure that it is possible).

It was some time ago, and not related to current version. All right I remove it.

Reply:
Can we make this change in the other `setup.py` files as well?